src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy

```diff
@@ -406,6 +411,13 @@ class ModuleDistribution extends DefaultTask
                     include(name: "*")
                 }
 
+                // WEB-INF/lib (*.jar)
+                zipfileset(dir: moduleDir,
+                        prefix: "WEB-INF",
+                        erroronmissingdir: false) {
+                    include(name: "lib/*.jar")
+                }
+
```

Comment:
Is this needed for the WAR file build?

Reply:
Yes, the jar files are expected to be in the WEB-INF/lib directory.